packages/dx-react-grid/src/plugins/virtual-table/virtual-table-state.tsx

```diff
@@ -16,7 +16,7 @@ const rowsComputed = ({ virtualRows }: Getters) => plainRows(virtualRows);
 const loadedRowsStartComputed = ({ virtualRows }: Getters) => loadedRowsStart(virtualRows);
 
 // tslint:disable-next-line: max-line-length
-class VirtualTableStateBase extends React.PureComponent<VirtualTableStateProps, VirtualTableStateState> {
+class VirtualTableStateBase extends React.PureComponent<VirtualTableStateProps, any> {
```

Comment:
- The `VirtualTableStateState` type can be removed from `virtula-table-state.types.ts` file. 
- Should be there `any` state? Maybe `DeprecatedLazyLoadingStrategyState`?

Reply:
`VirtualTableStateState` is the one we need. `DeprecatedLazyLoadingStrategyState` is from another PR :)